pom.xml

```diff
@@ -215,7 +215,7 @@
         <jboss-logging.version>3.3.1.Final</jboss-logging.version>
         <jcodec.version>0.2.0_1</jcodec.version>
         <jdom.bundle.version>1.1_4</jdom.bundle.version>
-        <jetty.solr.version>9.4.19.v20190610</jetty.solr.version>
```

Comment:
❓ Why do we use a separate Jetty version for Solr? As far as I can tell, Solr just ends up using the version that's provided by Karaf

Reply:
Solr now runs outside of DDF.  This jetty version is used for tests that start the `MiniSolrCloudCluster` which requires the Solr version of Jetty to be available to that JUnit class.